### PR TITLE
chore(ci): Explicit Hex2Byte Grep Check

### PR DESCRIPTION
**Pull request by refcell**
**Description**

Adds an explicit grep check to the `op-challenger` for demonstrative purposes that https://github.com/ethereum-optimism/optimism/pull/9283 covers the issue.

### CODE REVIEW

**Comment by refcell** [comment]
Curious if this would work.

**Comment by refcell** [comment]
(the `op-conductor` change is included since this branch was based on `develop` before rebasing)

File: op-challenger/Makefile [hunk 1]
```diff
@@ -9,21 +9,24 @@ LDFLAGS := -ldflags "$(LDFLAGSSTRING)"
...
+.PHONY: op-challenger
 
 fuzz:
 	go test -run NOTAREALTEST -v -fuzztime 10s -fuzz FuzzKeccak ./game/keccak/matrix
+.PHONY: fuzz
 
 clean:
 	rm bin/op-challenger
+.PHONY: clean
+
```
Comment (tynes) [inline op-challenger/Makefile:21]: This should already be covered by semgrep across whole codebase no?

Comment (trianglesphere) [reply op-challenger/Makefile:21]: Yea, it is covered by semgrep right now

**Pull request closed without merging**